include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -100,8 +100,36 @@ struct CollectiveMma<MainloopIntelXeXMX16<Stages, Schedule>, TileShape_, Element
   static constexpr auto Num_SGs = ATOM_N * ATOM_M * ATOM_K;
   static constexpr uint32_t MaxThreadsPerBlock = size(TiledMma{});
 
-  using Copy_A = typename Copy_Traits<GmemTiledCopyA, StrideA>::template DefaultTiledCopy<ElementA>;
-  using Copy_B = typename Copy_Traits<GmemTiledCopyB, StrideB>::template DefaultTiledCopy<ElementB>;
+  // Helper struct to deduce CopyOpA type
+  template<class EA, class SA, class TM, class GTCA>
+  struct CopyOpAHelper {
+    static auto get() {
+      auto tmp = make_tensor(make_gmem_ptr(static_cast<EA const*>(nullptr)),
+                             make_layout(make_shape(Int<BLK_M>{}, Int<BLK_K>{}, Int<1>{}), SA{}));
+      if constexpr (!std::is_void_v<GTCA>) {
+        return make_block_2d_copy_A(GTCA{}, TM{}, tmp(_,_,0));
+      } else {
+        return make_block_2d_copy_A(TM{}, tmp(_,_,0));
+      }
+    }
+  };
+
+  // Helper struct to deduce CopyOpB type
+  template<class EB, class SB, class TM, class GTCB>
+  struct CopyOpBHelper {
+    static auto get() {
+      auto tmp = make_tensor(make_gmem_ptr(static_cast<EB const*>(nullptr)),
+                             make_layout(make_shape(Int<BLK_N>{}, Int<BLK_K>{}, Int<1>{}), SB{}));
+      if constexpr (!std::is_void_v<GTCB>) {
+        return make_block_2d_copy_B(GTCB{}, TM{}, tmp(_,_,0));
+      } else {
+        return make_block_2d_copy_B(TM{}, tmp(_,_,0));
+      }
+    }
+  };
+
+  using CopyOpA = decltype(CopyOpAHelper<ElementA, StrideA, TiledMma, GmemTiledCopyA>::get());
+  using CopyOpB = decltype(CopyOpBHelper<ElementB, StrideB, TiledMma, GmemTiledCopyB>::get());
```

Comment:
Naming nit: these are not copy operations but TiledCopy instances (which wrap a copy operation with tiling information for the atom).
```suggestion
  using TiledCopyA = decltype(CopyOpAHelper<ElementA, StrideA, TiledMma, GmemTiledCopyA>::get());
  using TiledCopyB = decltype(CopyOpBHelper<ElementB, StrideB, TiledMma, GmemTiledCopyB>::get());
```